monkey/infection_monkey/credential_store/credentials_store.py

```diff
@@ -0,0 +1,29 @@
+from typing import Mapping
+
+from .i_credentials_store import ICredentialsStore
+
+
+class CredentialsStore(ICredentialsStore):
+    def __init__(self, credentials: Mapping = None):
+        self.stored_credentials = credentials
+
+    def add_credentials(self, credentials_to_add: Mapping) -> None:
+        if self.stored_credentials is None:
+            self.stored_credentials = {}
```

Comment:
This should never be the case if `self.stored_credentials` is properly initialized in the constructor.